### PR TITLE
added action for greeting to all

### LINKED ISSUE #924: URGENT: Add/Update a greetings github action.
# Description

Update/Add a new Github action for greeting purposes.

| name            | problem/motivation                                                  | possible solution  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| GitHub Action for Greetings | Current Greeting action can be far better | Explained Below | up for grabs |

**Possible Solution**

The current greetings action bot only triggers whenever a new contributor makes a new Issue/PR, we no longer need that, with the increasing contributor retention, I believe we can change the action to be triggered on every new  Issue/PR.

**Describe alternatives you've considered**

There is multiple actions/bot that one can play with for this case.

**Additional context**

The bot should actually print a meaningful message, link to contribution and styling guidelines is a must, but if you can think of something more creative, feel free to suggest.

### PR DESCRIPTION
**Pull request by Sukriti-sood**
Fixes #924 

Added greeting.yml .
Github Bot will Comment on Opening of  Issues and PRs 
Also for merging of PRs

### CODE REVIEW

**Comment by vybhav72954** [comment]
Provide a screenshot

**Comment by Sukriti-sood** [comment]
> Provide a screenshot

Screenshot from 2021-05-03 14-15-09

Screenshot from 2021-05-03 13-56-57

File: .github/workflows/greeting.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Create comment for issue
+      if: github.event_name =='issues'
+      uses: peter-evans/create-or-update-comment@v1
+      with:
+        issue-number: ${{tojson(github.event.issue.number)}}
+        body: |
+            @${{ github.actor }} Thanks for opening the New Issue, This Will definitely Improve our Project.
```
Comment (vybhav72954) [inline .github/workflows/greeting.yml:19]: Please append this line as well:
```Use /assign to self assign this issue```

Preferably in bolds

**Review by vybhav72954 — CHANGES_REQUESTED** [review]

File: .github/workflows/greeting.yml (hunk 1 above)
Comment (Sukriti-sood) [reply .github/workflows/greeting.yml:19]: Ook

**Comment by Sukriti-sood** [comment]
8182824-ad929979f1fff207261e725101fedeee

**Review by vybhav72954 — CHANGES_REQUESTED** [review]
One final chnage :P

That `/assign` line. Please put it into a newline.

**Comment by Sukriti-sood** [comment]
@vybhav72954  Fine.......?

fga · Issue #33

**Review by vybhav72954 — APPROVED** [review]

**Comment by vybhav72954** [comment]
LOL XD

**Pull request merged**